api/src/test/java/keywhiz/api/automation/v2/CreateSecretRequestV2Test.java

```diff
@@ -35,6 +36,12 @@
       .groups("secret-group1", "secret-group2")
       .build();
 
+  @Test
+  public void deserializesOwner() throws Exception {
+    CreateSecretRequestV2 request = fromJson(jsonFixture("fixtures/v2/createSecretRequestWithOwner.json"), CreateSecretRequestV2.class);
+    assertEquals("secret-owner", request.owner());
+  }
```

Comment:
Out of sheer paranoia I wanted to have a test to prove that JSON de-serialization wasn't the issue.